echo/server/dembrane/api/chat.py

```
@@ -449,7 +449,7 @@ async def post_chat(
                 get_transcripts=True,
                 top_k=top_k,
             )
-            logger.info(f"***rag_prompt: {rag_prompt}")
+            logger.info(f"rag_prompt: {rag_prompt}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)